### ISSUE TITLE
The slug is not translated

### ISSUE THREAD

**Issue by ko31** [ko31/simple-slug-translate#30]
When I create a post with the following steps, the slug is not translated.

1. Add a new post
2. Enter a title
3. Publish it immediately, the slug is not translated

**Comment by jeffwudev** [comment]
same issue

**Comment by jeffwudev** [comment]
wow, greate job. I found that adding an add_filter to the plugins_loaded function can also fix this issus, but I don't know if this is correct.
`add_filter( 'sanitize_title', array($this, 'sanitize_title'), 1 );`
```php
public function sanitize_title($title) {
    return $this->call_translate($title);
}
```

**Comment by ko31** [comment]
@jeff-wu 
I See! That method can also be converted. It depends on how you use it.
For example, it may not suit those who do not want to overwrite.

**Comment by ko31** [comment]
Fixed in version 2.6.1.
